test/common/test/api_utils_test.py

```diff
@@ -0,0 +1,203 @@
+"""
+Copyright 2017, EMC, Inc.
+"""
+import sys
+import subprocess
+
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test")
+
+import unittest
+
+#CIT
+from on_http_api2_0 import ApiApi as Api
+from config.api1_1_config import config as config_old   #TODO remove when 2.0 worklfow API is implemented
+from config.api2_0_config import config
+from json import loads
+from common.api_utils import api_node_select_from_config, api_node_select, api_validate_node_pollers
+
+from nose.plugins.attrib import attr
+import flogging
+logs = flogging.get_loggers()
+
+@attr(all=True, regression=True, smoke=True)
+
+class NodesTests(unittest.TestCase):
+    def setUp(self):
+        self.__client_old = config_old.api_client
+        self.__client = config.api_client
+    def __get_data(self):
+        return loads(self.__client.last_response.data)
+
+    def test_type_list(self):
+        """validates test utility method api_node_select() using an obm mac address will return the correct node id"""
```

Comment:
Nit:  nose/unittest doesn't parse the triple ticks when we do a list of the tests in the file.
Add a # to the comment directly under the def test_xxxx and rest of tests below.
(oh...unless you can fix unittest for me.....)  :-)
Otherwise, it displays like this:
```
run_tests.py -stack 13 -test test/api_utils_test.py  -list
*** Created config file: config/generated/fit-config-20170202-000127
*** Using config file: config/generated/fit-config-20170202-000127

Test Listing for: test/api_utils_test.py 
----------------------------------------------------------------------
*** Reloading config file: config/generated/fit-config-20170202-000127
2017-02-02 00:01:28,550 INFO    removing previous logging dir /emc/hohene/sandbox-github/rackhd/stu-rackhd/test/log_output/run_2017-02-01_23:00:10.d 28274 MainProcess stream-monitor/flogging/infra_logging.py:__init__@119 gl-main
2017-02-02 00:01:28,551 INFO    this runs logging dir /emc/hohene/sandbox-github/rackhd/stu-rackhd/test/log_output/run_2017-02-02_00:01:28.d 28274 MainProcess stream-monitor/flogging/infra_logging.py:__init__@119 gl-main
validates test utility method api_node_select_from_config() using obm mac address will return the correct node id ... ok
validates test utility method api_node_select_from_config() will return the correct nodes with valid obm settings ... ok
validates test utility method api_node_select() outputs nodes with validate obm settings ... ok
validates test utility method api_node_select() detects invalid sku type ... ok
validates test utility method api_node_select() detects if a provided node id is valid ... ok
validates test utility method api_node_select() using node id will validate node obm settings ... ok
validates test utility method api_node_select() using obm mac address will return the correct node id ... ok
validates test utility method api_validate_node_pollers() will return nodes with all active pollers ... ok
validates test utility method api_validate_node_pollers() will detect pollers for invalid node ... ok
validates test utility method api_validate_node_pollers() will return nodes with one active poller ... ok
Validates test utility method api_node_select() outputs all nodes for a provided sku name ... ok
validates test utility method api_node_select() using an obm mac address will return the correct node id ... ok

----------------------------------------------------------------------
Ran 12 tests in 0.009s
```